packages/data-models/flowTypes.ts

```diff
@@ -319,6 +319,7 @@ export namespace FlowTypes {
     | "audio_pause"
     | "audio_end"
     | "audio_first_start"
+    | "accept_terms"
```

Comment:
I think this is a bit too content-specific to be an action type, probably better to just use `set_field` action and manage via fields